var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -118,7 +175,7 @@ def configure_args(self):
         options = [
             '--libdir={0}'.format(prefix.lib64),
             '--disable-multilib',
-            '--enable-languages={0}'.format(','.join(enabled_languages)),
+            '--enable-languages={0}'.format(spec.variants['languages'].value),
```

Comment:
Not sure if this is actually the appropriate syntax. I'll test it out once the GMP website comes back up.